### DISCUSSION TITLE
Livewire Pagination broken with Laravel 9.3.8 update ?

### DISCUSSION THREAD

**Discussion by dominiquevilain** [livewire/livewire#5324] [7. Bugs (archived)]
Livewire Pagination seems to be broken after the recent laravel release (9.3.8) with its new Blade engine.

The report is described here : https://github.com/livewire/livewire/discussions/5323

**Comment by joshhanley** [comment]
@dominiquevilain please don't open two discussions on the same issue, instead please just update the original one 🙂 we will investigate in #5323.
